src/browser/starter.js

```diff
@@ -698,16 +698,29 @@ V86Starter.prototype.run = async function()
  */
 V86Starter.prototype.stop = async function()
 {
-    this.bus.send("cpu-stop");
+    if (!this.cpu_is_running) {
+        return;
+    }
```

Comment:
Preserve the style of the other code (remove space after `if`, put `{` on its own line).